CardScan/CardScan/Classes/ScanStats.swift

```diff
@@ -63,6 +63,19 @@ public struct ScanStats {
                 "user_canceled": self.userCanceled]
     }
     
+    func createPayload() -> ScanStatisticsPayload {
+        let scanStats = ScanStatistics(scans: self.scans,
+                                       cardsDetected: self.cardsDetected,
+                                       torchOn: self.torchOn,
+                                       orientation: self.orientation,
+                                       success: self.success,
+                                       duration: self.duration(),
+                                       model: self.model,
+                                       permissionGranted: self.permissionGranted,
+                                       userCanceled: self.userCanceled)
```

Comment:
maybe this is iOS style, but I really don't like hanging indents like this. It's not guaranteed to be a multiple of four and can break easily by changing anything in the first line. instead, I greatly prefer the style:
```
let scanStats = ScanStatistics(
    scans: self.scans,
    ...
    userCanceled: self.userCanceled,
)
```

Reply:
I think it depends on the style guide of each place. I just looked different companies guides on multi-line styling and a lot of them just use the xcode indentation. But it's not that big of a deal right now so I can change it to how you suggest it :-)